ambari-infra-manager-it/pom.xml

```diff
@@ -182,6 +211,53 @@
         <directory>src/test/resources</directory>
       </testResource>
     </testResources>
+    <plugins>
+      <plugin>
+        <groupId>io.swagger</groupId>
```

Comment:
are we sure we want to generate this with every build? im fine if that would require some property to generate it